blosc/blosc2.c

```diff
@@ -2270,18 +2279,6 @@ int blosc_compress_context(blosc2_context* context) {
     }
   }
 
-  int dont_split = (context->header_flags & 0x10) >> 4;
-  int nstreams = context->nblocks;
-  if (!dont_split) {
-    // When splitting, the number of streams is computed differently
-    if (context->leftover) {
-      nstreams = (context->nblocks - 1) * context->typesize + 1;
-    }
-    else {
-      nstreams *= context->typesize;
-    }
-  }
-
```

Comment:
This code should get back to its original position.

Reply:
solved at f2fdffa71814dd1cbda6d3388e551c7e61214f64